test/controllers/concerns/authentication_concern_test.rb

```diff
@@ -0,0 +1,18 @@
+require 'test_helper'
+
+class AuthenticationConcernTest < ActionController::TestCase
+  setup do
+    @controller = SessionsController.new
+  end
+
+  def test_unauthenticate_user
+    # should disconnect all the connections established for current users
+    user = User.create(name: 'Sticky Fingaz')
+
+    post :create, params: { user_id: user.id }
+
+    ActionCable.server.expects(:disconnect).with(has_entries(current_user: user))
+
+    delete :destroy
+  end
+end
```

Comment:
I don't think we really need this test. It just tests that we call a mock we already established ourselves. Let's nix it. Then you can nix the mocha inclusion as well and just have the #disconnect call.